proxy/envoy/policy.go

```diff
@@ -61,6 +61,7 @@ func insertMixerFilter(listeners []*Listener, instances []*model.ServiceInstance
 							"source.ip":  context.IPAddress,
 							"source.uid": context.UID,
 						},
+						QuotaName: "RequestCount",
```

Comment:
To my understanding, `RequestCount` is a specifc configured quota. This looks like the proxy will be telling mixer to always use that limit (in our case, 5 qps).

I would expect that there would be some scheme for mixerclient to discover the set of quotas and then optionally decide which quota name to request in its Quota call (and/or have that done entirely on the mixer side).